src/TwigComponent/src/ComponentFactory.php

```diff
@@ -71,6 +71,10 @@ public function create(string $name, array $data = []): MountedComponent
 
         // ensure remaining data is scalar
         foreach ($data as $key => $value) {
+            if ($value instanceof \Stringable) {
```

Comment:
Funny timing - we JUST did this with `stimulus_controller()` also. There, we went further and checked for `__toString()`: https://github.com/symfony/webpack-encore-bundle/pull/171/files#diff-aac7ca1aa215ab99c76587c963238c082fb42b7a4315344002b7df3046306b8fR64

Reply:
I don't believe we need to check for the method because this lib is PHP 8+.

Reply:
Good catch!